sstables/index_reader.hh

```diff
@@ -1036,6 +1036,24 @@ public:
         return cur_bsearch->last_block_offset();
     }
 
+    // Return position_range on the first partition which is not smaller than pos.
+    // Must be called for non-decreasing positions.
```

Comment:
the signature has `key` as parameter, not `pos`

Reply:
Will fix. thanks